bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/component.jsx

```diff
@@ -622,124 +570,197 @@ class PresentationUploader extends Component {
+
+    let hasNewUpload = false;
 
-    let awaitingConversion = false;
-    presentations.map((presentation) => {
-      if (!presentation.conversion.done) awaitingConversion = true;
-      return null;
+    presentations.map((item) => {
+      if (item.id.indexOf(item.filename) !== -1 && item.upload.progress === 0) hasNewUpload = true;
     });
 
-    const confirmLabel = awaitingConversion
-      ? intl.formatMessage(intlMessages.uploadLabel)
-      : intl.formatMessage(intlMessages.confirmLabel);
+    return isOpen ? (
+      <div className={styles.modal}>
+        <div
+          className={styles.modalInner}
+        >
+          <div className={styles.modalHeader}>
+            <h1>Presentation</h1>
+            <div className={styles.actionWrapper}>
+              <Button
+                className={styles.dismiss}
+                color="default"
+                onClick={handleDismiss}
+                label={intl.formatMessage(intlMessages.dismissLabel)}
+                aria-describedby={intl.formatMessage(intlMessages.dismissDesc)}
+              />
+              <Button
+                className={styles.confirm}
+                color="primary"
+                onClick={() => this.handleConfirm(hasNewUpload)}
+                disabled={disableActions}
+                label={hasNewUpload
+                  ? intl.formatMessage(intlMessages.uploadLabel)
+                  : intl.formatMessage(intlMessages.confirmLabel)
+                }
+              />
+            </div>
+          </div>
 
-    return (
-      <ModalFullscreen
-        title={intl.formatMessage(intlMessages.title)}
-        preventClosing={preventClosing}
-        confirm={{
-          callback: this.handleConfirm,
-          label: confirmLabel,
-          description: intl.formatMessage(intlMessages.confirmDesc),
-          disabled: disableActions,
-        }}
-        dismiss={{
-          callback: this.handleDismiss,
-          label: intl.formatMessage(intlMessages.dismissLabel),
-          description: intl.formatMessage(intlMessages.dismissDesc),
-          disabled: disableActions,
-        }}
-      >
-        <p>{intl.formatMessage(intlMessages.message)}</p>
-        {this.renderPresentationList()}
-        <div className={styles.dropzoneWrapper}>
+          <div className={styles.modalHint}>
+            {`${intl.formatMessage(intlMessages.message)}`}
+          </div>
+          {this.renderPresentationList()}
           {isMobileBrowser ? this.renderPicDropzone() : null}
           {this.renderDropzone()}
         </div>
-      </ModalFullscreen>
-    );
+      </div>
+    ) : null;
   }
 }
 
-PresentationUploader.propTypes = propTypes;
-PresentationUploader.defaultProps = defaultProps;
+PresentationUploader .propTypes = propTypes;
+PresentationUploader .defaultProps = defaultProps;
 
-export default withModalMounter(injectIntl(PresentationUploader));
+export default PresentationUploader ;
```

Comment:
make sure to run eslint before committing
```suggestion
export default PresentationUploader;
```